mlflow/keras.py

```diff
@@ -82,9 +82,13 @@ def save_model(keras_model, path, conda_env=None, mlflow_model=Model(), custom_o
                                 'tensorflow=1.8.0'
                             ]
                         }
-    :param keras_module: Keras module to be used to save / load the model. If not provided, MLflow
-    will attempt to infer the Keras module based on the given model.
     :param mlflow_model: MLflow model config this flavor is being added to.
+    :param custom_objects: Keras ``custom_objects`` dictionary containing String:Class
```

Comment:
nit: `String:Class` is a bit confusing to me. Maybe we could make it something like `str to classobj` for the real type names?

Reply:
I've updated it to be the description that is used in the keras repository for the custom objects object.